python/src/com/google/idea/blaze/python/resolve/provider/AbstractPyImportResolverStrategy.java

```diff
@@ -80,6 +80,16 @@ public final void addImportCandidates(
       PsiFile file = psi.getContainingFile();
       if (file != null) {
         quickFix.addImport(psi, file, candidate.removeLastComponent());
+        continue;
```

Comment:
(HT @brendandouglas)

Please use `PyUtils.turnDirIntoUtil(resolver.get(psiManager))` instead, so these entire diff is not needed.